src/ledger/GeneralizedLedgerEntry.h

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#pragma once
+
+#include "xdr/Stellar-ledger-entries.h"
+#include "xdr/Stellar-ledger.h"
+
+namespace stellar
+{
+
+enum class GeneralizedLedgerEntryType
```

Comment:
maybe just `Internal` instead of `Generalized`?

Reply:
`Internal` seems better. I'm going to hold off on changing this until everything else is settled here, then global-search-and-replace.